INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
INFO:root:prepare training...
INFO:root:starting training...
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled
	 [[Node: valid_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](valid_data_queue/queue, _arg_valid_data_queue/E_0_1, _arg_valid_data_queue/Ea_0_2, _arg_valid_data_queue/F_0_3, _arg_valid_data_queue/Z_0_7, _arg_valid_data_queue/D_0_0, _arg_valid_data_queue/Q_0_4, _arg_valid_data_queue/Qa_0_5, _arg_valid_data_queue/R_0_6, _arg_valid_data_queue/idx_i_0_9, _arg_valid_data_queue/idx_j_0_10, _arg_valid_data_queue/batch_seg_0_8)]]

Caused by op 'valid_data_queue/queue_enqueue', defined at:
  File "train.py", line 131, in <module>
    valid_queue = DataQueue(data_provider.next_valid_batch, capacity=args.num_valid//args.valid_batch_size, scope="valid_data_queue")
  File "/home/wahmed/codes/dompe/PhysNet/training/DataQueue.py", line 31, in __init__
    self._enqueue_op = self.queue.enqueue(placeholders)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/data_flow_ops.py", line 339, in enqueue
    self._queue_ref, vals, name=scope)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 3977, in queue_enqueue_v2
    timeout_ms=timeout_ms, name=name)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

CancelledError (see above for traceback): Enqueue operation was cancelled
	 [[Node: valid_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](valid_data_queue/queue, _arg_valid_data_queue/E_0_1, _arg_valid_data_queue/Ea_0_2, _arg_valid_data_queue/F_0_3, _arg_valid_data_queue/Z_0_7, _arg_valid_data_queue/D_0_0, _arg_valid_data_queue/Q_0_4, _arg_valid_data_queue/Qa_0_5, _arg_valid_data_queue/R_0_6, _arg_valid_data_queue/idx_i_0_9, _arg_valid_data_queue/idx_j_0_10, _arg_valid_data_queue/batch_seg_0_8)]]

INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
INFO:root:prepare training...
INFO:root:starting training...
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled
	 [[Node: train_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](train_data_queue/queue, _arg_train_data_queue/E_0_1, _arg_train_data_queue/Ea_0_2, _arg_train_data_queue/F_0_3, _arg_train_data_queue/Z_0_7, _arg_train_data_queue/D_0_0, _arg_train_data_queue/Q_0_4, _arg_train_data_queue/Qa_0_5, _arg_train_data_queue/R_0_6, _arg_train_data_queue/idx_i_0_9, _arg_train_data_queue/idx_j_0_10, _arg_train_data_queue/batch_seg_0_8)]]

Caused by op 'train_data_queue/queue_enqueue', defined at:
  File "train.py", line 130, in <module>
    train_queue = DataQueue(data_provider.next_batch, capacity=1000, scope="train_data_queue")
  File "/home/wahmed/codes/dompe/PhysNet/training/DataQueue.py", line 31, in __init__
    self._enqueue_op = self.queue.enqueue(placeholders)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/data_flow_ops.py", line 339, in enqueue
    self._queue_ref, vals, name=scope)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 3977, in queue_enqueue_v2
    timeout_ms=timeout_ms, name=name)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

CancelledError (see above for traceback): Enqueue operation was cancelled
	 [[Node: train_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](train_data_queue/queue, _arg_train_data_queue/E_0_1, _arg_train_data_queue/Ea_0_2, _arg_train_data_queue/F_0_3, _arg_train_data_queue/Z_0_7, _arg_train_data_queue/D_0_0, _arg_train_data_queue/Q_0_4, _arg_train_data_queue/Qa_0_5, _arg_train_data_queue/R_0_6, _arg_train_data_queue/idx_i_0_9, _arg_train_data_queue/idx_j_0_10, _arg_train_data_queue/batch_seg_0_8)]]

INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
INFO:root:prepare training...
INFO:root:starting training...
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled
	 [[Node: train_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](train_data_queue/queue, _arg_train_data_queue/E_0_1, _arg_train_data_queue/Ea_0_2, _arg_train_data_queue/F_0_3, _arg_train_data_queue/Z_0_7, _arg_train_data_queue/D_0_0, _arg_train_data_queue/Q_0_4, _arg_train_data_queue/Qa_0_5, _arg_train_data_queue/R_0_6, _arg_train_data_queue/idx_i_0_9, _arg_train_data_queue/idx_j_0_10, _arg_train_data_queue/batch_seg_0_8)]]

Caused by op 'train_data_queue/queue_enqueue', defined at:
  File "train.py", line 130, in <module>
    train_queue = DataQueue(data_provider.next_batch, capacity=1000, scope="train_data_queue")
  File "/home/wahmed/codes/dompe/PhysNet/training/DataQueue.py", line 31, in __init__
    self._enqueue_op = self.queue.enqueue(placeholders)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/data_flow_ops.py", line 339, in enqueue
    self._queue_ref, vals, name=scope)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 3977, in queue_enqueue_v2
    timeout_ms=timeout_ms, name=name)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/wahmed/miniconda3/envs/physnet/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

CancelledError (see above for traceback): Enqueue operation was cancelled
	 [[Node: train_data_queue/queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT32, DT_INT32, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](train_data_queue/queue, _arg_train_data_queue/E_0_1, _arg_train_data_queue/Ea_0_2, _arg_train_data_queue/F_0_3, _arg_train_data_queue/Z_0_7, _arg_train_data_queue/D_0_0, _arg_train_data_queue/Q_0_4, _arg_train_data_queue/Qa_0_5, _arg_train_data_queue/R_0_6, _arg_train_data_queue/idx_i_0_9, _arg_train_data_queue/idx_j_0_10, _arg_train_data_queue/batch_seg_0_8)]]

INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
INFO:root:prepare training...
INFO:root:starting training...
INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:53: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:55: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:58: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:59: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:69: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/DenseLayer.py:17: The name tf.add_to_collection is deprecated. Please use tf.compat.v1.add_to_collection instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/DenseLayer.py:17: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:102: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:284: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

INFO:root:prepare training...
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/DataQueue.py:17: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/DataQueue.py:30: The name tf.PaddingFIFOQueue is deprecated. Please use tf.queue.PaddingFIFOQueue instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/RBFLayer.py:52: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/activation_fn.py:21: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/InteractionLayer.py:61: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/InteractionLayer.py:68: The name tf.segment_sum is deprecated. Please use tf.math.segment_sum instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:10: The name tf.train.exponential_decay is deprecated. Please use tf.compat.v1.train.exponential_decay instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:16: The name tf.global_norm is deprecated. Please use tf.linalg.global_norm instead.

WARNING:tensorflow:From /home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/training/moving_averages.py:433: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:30: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:48: The name tf.assign is deprecated. Please use tf.compat.v1.assign instead.

WARNING:tensorflow:From train.py:461: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.

WARNING:tensorflow:From train.py:462: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

WARNING:tensorflow:From train.py:462: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From train.py:571: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

WARNING:tensorflow:From train.py:585: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.

INFO:root:starting training...
WARNING:tensorflow:From train.py:454: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.

INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.CancelledError'>, Enqueue operation was cancelled
	 [[node train_data_queue/queue_enqueue (defined at /home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py:1748) ]]

Original stack trace for 'train_data_queue/queue_enqueue':
  File "train.py", line 275, in <module>
  File "/home/wahmed/codes/dompe_workspace/PhysNet/training/DataQueue.py", line 31, in __init__
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/ops/data_flow_ops.py", line 346, in enqueue
    self._queue_ref, vals, name=scope)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/ops/gen_data_flow_ops.py", line 4410, in queue_enqueue_v2
    timeout_ms=timeout_ms, name=name)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/framework/op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "/home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/framework/ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()

INFO:root:creating directories...
INFO:root:writing args to file...
INFO:root:loading dataset...
INFO:root:creating neural network...
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:53: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:55: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:58: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:59: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:69: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/DenseLayer.py:17: The name tf.add_to_collection is deprecated. Please use tf.compat.v1.add_to_collection instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/DenseLayer.py:17: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:102: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/NeuralNetwork.py:284: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

INFO:root:prepare training...
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/DataQueue.py:17: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/DataQueue.py:30: The name tf.PaddingFIFOQueue is deprecated. Please use tf.queue.PaddingFIFOQueue instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/RBFLayer.py:52: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/activation_fn.py:21: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/InteractionLayer.py:61: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/neural_network/layers/InteractionLayer.py:68: The name tf.segment_sum is deprecated. Please use tf.math.segment_sum instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:10: The name tf.train.exponential_decay is deprecated. Please use tf.compat.v1.train.exponential_decay instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:16: The name tf.global_norm is deprecated. Please use tf.linalg.global_norm instead.

WARNING:tensorflow:From /home/wahmed/.local/lib/python3.6/site-packages/tensorflow_core/python/training/moving_averages.py:433: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:30: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From /home/wahmed/codes/dompe_workspace/PhysNet/training/Trainer.py:48: The name tf.assign is deprecated. Please use tf.compat.v1.assign instead.

WARNING:tensorflow:From train.py:461: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.

WARNING:tensorflow:From train.py:462: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

WARNING:tensorflow:From train.py:462: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From train.py:571: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

WARNING:tensorflow:From train.py:585: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.

INFO:root:starting training...
WARNING:tensorflow:From train.py:454: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.

